NodeJS/CLI Password Generator/README.md

```diff
@@ -0,0 +1,51 @@
+# CLI Password Generator
+
+## Description 
+Command line app to generate random passwords. This uses crypto module due to which generated passwords are totally random, and unpredictable.
+
+## Tech Stack
+
+NodeJS 
+
+## Demonstration
+
+![Password Generator](utils/passgen.png)
+
+## Usage
+
+Install dependencies
+
+```
+npm install
+```
+
+Run file
+
+```
+node index (options)
+```
+
+To create a symlink to run "passgen" from anywhere
+
+```
+npm link
+
+# Now you can run
+passgen (options)
+
+# To remove symlink
+npm unlink 
+or
+npm uninstall -g passgen (to completely remove the package)
+```
```

Comment:
mention in these way that looks good.
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/156782741-acc41d6b-2a08-4765-8026-95ca3f40ac3d.png">